ui/components/component-library/avatar-network/avatar-network.stories.js

```diff
@@ -75,12 +75,12 @@ export const Size = (args) => (
   </Box>
 );
 
-export const NetworkName = Template.bind({});
-NetworkName.args = {
-  networkImageUrl: '',
+export const Name = Template.bind({});
+Name.args = {
+  src: '',
 };
 
-export const NetworkImageUrl = Template.bind({});
+export const Src = Template.bind({});
```

Comment:
For this story can we add all the popular network images 

<img width="932" alt="Screenshot 2022-11-23 at 12 38 04 PM" src="https://user-images.githubusercontent.com/8112138/203642856-a4ef7bde-81ca-4b15-88b2-6b7b6bfe8899.png">